satellite/peer.go

```diff
@@ -360,6 +361,7 @@ func New(log *zap.Logger, full *identity.FullIdentity, db DB, revocationDB exten
 	{ // setup contact service
 		log.Debug("Setting up contact service")
 		peer.Contact.Service = contact.NewService(peer.Log.Named("contact"), peer.Overlay.Service, peer.Transport)
+		peer.Contact.Endpoint = contact.NewEndpoint(peer.Log.Named("contact"), peer.Contact.Service)
```

Comment:
in other places, we differentiate the logs for the service and the endpoint, to be able to tell which one logged a thing. like `s/contact/contact:endpoint/` in this case.